ページを完成させる

caramelribon opened the pull request:
branchを切らずに変更していまったので、プルリクエストができなかったため、ブランチを切ったうえでもう一回コミットしました

## セルフチェック
- [x] お手本通りの見た目であるか　ホバー時の挙動、レスポンシブ対応など要確認
- [x] ブランチ名、ファイル名、クラス名、インデントなど、コーディング規則を逸脱していないか
- [x] 記述の重複はないか

作成したサイトのURL
[URL](https://caramelribon.github.io/wakana_tatsuta-frontend-basic-last-assignment/)

caramelribon committed:
pref:一回目のコミット

branchを切らずに変更していまったので、プルリクエストができなかったため、ブランチを切ったうえでもう一回コミットしました

caramelribon committed:
pref:imgファイルパスの変更

imgファイルをpagesに追加&パスの変更

Higashiji reviewed (commented):
![image](https://user-images.githubusercontent.com/83479175/187054531-13f56995-9aa7-4b9a-a56b-ca9ccf8cf48d.png)
[お手本サイト](https://demo.tcd-theme.com/tcd063/)のような感じで、背景とヘッダーが重なるようにしてください。

caramelribon committed:
pref:ヘッダーと背景画像が重なるようにする

ヘッダーと背景画像がかさなるように、topの方をrelative、headerの方をabsoluteにして、ヘッダーのレイアウトを崩さないようにtop:0px, width:100%を加えました

caramelribon commented:
お手本のサイトと同じになるようにヘッダーをトップの上に重ねました。

## セルフチェック
- [x] お手本通りの見た目であるか　ホバー時の挙動、レスポンシブ対応など要確認
- [x] ブランチ名、ファイル名、クラス名、インデントなど、コーディング規則を逸脱していないか
- [x] 記述の重複はないか

作成したサイトのURL
[URL](https://caramelribon.github.io/wakana_tatsuta-frontend-basic-last-assignment/)

Higashiji reviewed (approved):
LGTMです！
